tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -522,3 +522,189 @@ Feature: collaboration (wopi)
+    Given user "Alice" has created folder "testFolder"
+    When user "Alice" creates file "testfile.txt" inside folder "testFolder" in space "Personal" using wopi endpoint
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "file_id"
+        ],
+        "properties": {
+          "file_id": {
+            "type": "string",
+            "pattern": "^%file_id_pattern%$"
+          }
+        }
+      }
+      """
+    And as "Alice" file "testFolder/testfile.txt" should exist
+
+
+  Scenario Outline: sharee with permission Editor/Uploader creates text file inside shared folder
+    Given user "Alice" has created folder "testFolder"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | testFolder         |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    When user "Brian" creates file "testFile.txt" inside folder "testFolder" in space "Shares" using wopi endpoint
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "file_id"
+        ],
+        "properties": {
+          "file_id": {
+            "type": "string",
+            "pattern": "^%file_id_pattern%$"
+          }
+        }
+      }
+      """
+    And as "Alice" file "testFolder/testFile.txt" should exist
+    And as "Brian" file "Shares/testFolder/testFile.txt" should exist
+    Examples:
+      | permissions-role |
+      | Editor           |
+      | Uploader         |
+
+
+  Scenario: sharee with Viewer tries to creates text file inside shared folder
+    Given user "Alice" has created folder "testFolder"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | testFolder |
+      | space           | Personal   |
+      | sharee          | Brian      |
+      | shareType       | user       |
+      | permissionsRole | Viewer     |
+    When user "Brian" tries to creates file "testFile.txt" inside folder "testFolder" in space "Shares" using wopi endpoint
+    Then the HTTP status code should be "500"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "code",
+          "message"
+        ],
+        "properties": {
+          "code": {
+            "const": "SERVER_ERROR"
+          },
+          "message": {
+            "const": "error calling InitiateFileUpload"
+          }
+        }
+      }
+      """
+    And as "Alice" file "testFolder/testFile.txt" should not exist
+    And as "Brian" file "Shares/testFolder/testFile.txt" should not exist
+
+
+  Scenario: create text file in project space using wopi endpoint
```

Comment:
this should be covered by scenario below

Reply:
the scenario below is for the shared project space, where user with different share role creating file inside the shared project space.

Reply:
then we might need to mention space admin in this scenario